migration-guides/0.15-Ecosystem-Update.md

```diff
@@ -0,0 +1,1693 @@
+## `purescript` Libraries
+
+### purescript-arrays
+
+Breaking changes:
+- Migrate FFI to ES modules (#218 by @kl0tl and @JordanMartinez)
+- Drop deprecated `group'` and `empty` (#219 by @JordanMartinez)
+
+New features:
+
+Bugfixes:
```

Comment:
I think we should strip out empty sections

Reply:
For example, this section would read:

```
### purescript-arrays

Breaking changes:
- Migrate FFI to ES modules (#218 by @kl0tl and @JordanMartinez)
- Drop deprecated `group'` and `empty` (#219 by @JordanMartinez)

Other improvements:
- Fixed minor documentation issue with `find` (#216 by @JamieBallingall)
```

Reply:
> Could this be done easily in your script by trimming out sections that make it to the next section header without any contents besides newlines?

It should be possible so long as all repos are using the same general headers

Reply:
Done!

I also tried removing a lot of duplicate changelog entries and then removing entire libraries if that's all the changes that were made there. The latest commit shows this diff.